documentation/index.md

```diff
@@ -2,3 +2,5 @@
 template: home.html
 title: QField Ecosystem Documentation
 ---
+
+# QField Ecosystem Documentation
```

Comment:
NOTE if this is an empty file, it blows up for unknown reason. Decided to reuse a string that is already translated as a workaround.